test/tests/ExportListTest.php

```diff
@@ -13,7 +13,7 @@
 
 class ExportListTest extends SilverpopBaseTestClass {
   public function testExportList() {
-    $container = array();
+    $container = [];
```

Comment:
Any particular reason for this change? It was done in the old style to make the connector compatible with older versions of PHP, which didn't yet support the `[]` empty array notation. Not a blocker, I don't think, because very few clients are on versions below 5.4 any more.

Reply:
Oh right - yes - I guess I've been switching everything to be consistent on the new style now.